AWS/delete_iam_user.ipynb

```diff
@@ -0,0 +1,1021 @@
+{
```

Comment:
<p>Could we make posting of Slack message Optional? Meaning we can take the Slack channel as one of the Optional Parameter for the Runbook. If specified, then we can send the message. We can use the Action -&gt; start condition to check if the input channel is not <code> None</code> and send the message only when the input channel is given by the user.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/unskript/Awesome-CloudOps-Automation/pull/153/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
i think that could just be part of the configuration of the runbook - it is easy enough to remove that action. it will not harm the rest of the runbook.